src/styleeditorwindow.cpp

```diff
@@ -97,7 +97,7 @@ StyleEditorWindow::StyleEditorWindow(QWidget *parent) :
          childButton->installEventFilter(this);
     }
 
-    new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_S), this, SLOT(toggleWindowVisibility()));
+    new QShortcut(QKeySequence(Qt::CTRL, Qt::SHIFT, Qt::Key_S), this, SLOT(toggleWindowVisibility()));
```

Comment:
These do not mean the same thing. First there was a shortcut that you could trigger by holding Ctrl and Shift while pressing S, whereas now you have a shortcut that it triggered by pressing Ctrl, then Shift, then S, in that order (though I'm not sure such a shortcut actually works).

The change should have been to use `Qt::CTRL | Qt::SHIFT | Qt::Key_S` (which fortunately also works in Qt 5).